priv/templates/phx.gen.auth/registration_new.html.heex

```diff
@@ -7,20 +7,22 @@
     </div>
   <%% end %>
 
-  <%%= label f, :email %>
-  <%%= email_input f, :email, required: true %>
-  <%%= error_tag f, :email %>
+  <%%= label(f, :email) %>
+  <%%= email_input(f, :email, required: true) %>
+  <%%= error_tag(f, :email) %>
 
-  <%%= label f, :password %>
-  <%%= password_input f, :password, required: true %>
-  <%%= error_tag f, :password %>
+  <%%= label(f, :password) %>
+  <%%= password_input(f, :password, required: true) %>
+  <%%= error_tag(f, :password) %>
 
   <div>
-    <%%= submit "Register" %>
+    <%%= submit("Register") %>
   </div>
 </.form>
 
 <p>
-  <%%= link "Log in", to: Routes.<%= schema.route_helper %>_session_path(@conn, :new) %> |
```

Comment:
Wrapped `links` into `span` to instruct the formatter to keep the lines, otherwise it would try to keep everything inside the `p` element and would end up formatting as:

```heex
<p>
  <%= link("Log in", to: Routes.user_session_path(@conn, :new)) %> | <%= link(
    "Forgot your password?",
    to: Routes.user_reset_password_path(@conn, :new)
  ) %>
</p>
```

The second `span` breaks into a new line because it doesn't fit into the default length of 98 chars.